### PR TITLE
CoreDNS - Add 2 e2e jobs running kube-dns for comparisons

### PR DESCRIPTION
**Pull request by fturib**
## Ability to verify tests results of CoreDNS against kube-dns

CoreDNS is expected to replace kube-dns. See KEP : https://github.com/kubernetes/community/pull/1956

One of the graduation criteria is to be able to:

> Validate all e2e conformance and DNS related tests (xxx-kubernetes-e2e-gce, ci-kubernetes-e2e-gce-gci-ci-master and filtered by --ginkgo.skip=\\[Slow\\]|\\[Serial\\]|\\[Disruptive\\]|\\[Flaky\\]|\\[Feature:.+\\]) for CoreDNS. None of the tests successful with Kube-DNS should be failing with CoreDNS.

I propose here these 2 extra tests that will run always with kube-dns, whatever the default DNS service defined for K8s:
- the test for graduation criteria
- another test to validate the [Serial], [Slow] (but not Flaky). These tests includes some other DNS specifics (like updating the DNS configuration by configMap).

NOTE: we may remove these tests after the graduation.

### CODE REVIEW

**Comment by MrHohn** [comment]
Fixed https://github.com/kubernetes/dns/issues/227.

**Comment by MrHohn** [comment]
@fturib Thanks!

/lgtm

**Comment by MrHohn** [comment]
Holding this for now until https://github.com/kubernetes/kubernetes/pull/62147 goes in.

**Comment by fejta** [comment]
/lifecycle stale
Safe to close this one? Looks like the dependant PR is dead

**Comment by fturib** [comment]
I will present again these changes when it will be validated that CoreDNS is default for kube-up.

**Comment by rajansandeep** [comment]
@fturib 
This should be opened again as CoreDNS is now the default in kube-up as well.
/reopen

File: prow/config.yaml [hunk 1]
```diff
@@ -9948,6 +9948,19 @@ periodics:
       - --bare
       image: gcr.io/k8s-testimages/kubekins-e2e:v20180418-79a97ca02-master
 
+- interval: 4h
+  agent: kubernetes
+  name: ci-kubernetes-e2e-gci-gce-kube-dns
```
Comment (fejta) [inline prow/config.yaml]: These jobs will need to live in somewhere like test-infra/config/jobs/kubernetes/sig-foo/foo-config.yaml now.

Comment (MrHohn) [reply prow/config.yaml]: Probably use https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-network/sig-network-misc.yaml.

**Comment by rajansandeep** [comment]
This PR is old and seems a lot to rebase. I've opened https://github.com/kubernetes/test-infra/pull/9921 in favor of this PR.
/close

**Pull request closed without merging**